pa_config/rca_master.conf

```diff
@@ -109,20 +109,47 @@
   // Default Values are added in the config specific file as well
   // Modify them as well whenever changing the default values (DeciderConfig.java)
   "decider-config-settings": {
-    // Decreasing order of priority for the type of workload we can expect on the cluster.
-    // Priority order in the list goes from most expected to the lease expected workload type.
     "workload-type": {
-      "priority-order": ["ingest", "search"]
+      "prefer-ingest": true,
+      "prefer-search": true
```

Comment:
The other format seems much clearer to me. Why did we change it to this? What does this mean?

Reply:
Specifying the priority order as an array has two issues : 
1. we can't assign search and index operations with the same level of priority
2. we need maintain a function to parse all elements in array